worksheet_A/TerminalHacking.cpp

```diff
@@ -40,7 +54,34 @@ int main()
 	}
 
 	// TODO: implement the rest of the game
+	int lives = 5;
+	bool playerHasChosen;
+	std::string playerInput;
+	std::string guess;
+	int wordLikeness;
```

Comment:
Do all of these need to be declared outside the loop? It's generally good to keep variable declarations close to the point of use.